README.md

```diff
@@ -9,247 +9,128 @@ Also, see the [API documentation](https://aws.github.io/aws-encryption-sdk-c/htm
 
 [Security issue notifications](./CONTRIBUTING.md#security-issue-notifications)
 
-## Dependencies
+## Build recipes with KMS support
```

Comment:
Can we be very clear here? "Building the AWS Encryption SDK for C (with support for AWS KMS)"

(The current statement can be interpreted as using KMS to help with the build.)